Build fails when code coverage is enabled

Steps to reproduce:

- Clone https://github.com/IBM-Swift/Kitura-Sample
- Generate an Xcode project
- Enable code coverage for **KituraSampleRouter** in Edit Scheme > Test > Options.
- Test project

This fails with the following error:

```
Undefined symbols for architecture x86_64:
  "static (extension in Foundation):Swift.String._unconditionallyBridgeFromObjectiveC(__ObjC.NSString?) -> Swift.String", referenced from:
      __C.StringTransform.rawValue.getter : Swift.String in KituraStencilTemplateEngine.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Adding an explicit `import Foundation` to **KituraStencilTemplateEngine.swift** fixes it.

--- a/.travis.yml
+++ b/.travis.yml
@@ -30,7 +30,7 @@ matrix:
       sudo: required
       env: SWIFT_SNAPSHOT=4.0.3
     - os: osx
-      osx_image: xcode9.3beta
+      osx_image: xcode9.3
       sudo: required
 
 before_install:

--- a/Sources/KituraStencil/KituraStencilTemplateEngine.swift
+++ b/Sources/KituraStencil/KituraStencilTemplateEngine.swift
@@ -14,6 +14,7 @@
  * limitations under the License.
  **/
 
+import Foundation
 import KituraTemplateEngine
 import Stencil
 import PathKit

